src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -739,6 +739,67 @@ class NBDWatchCtx : public librbd::UpdateWatchCtx
   librados::IoCtx &io_ctx;
   librbd::Image &image;
   unsigned long size;
```

Comment:
As already done for `new_size`:
```suggestion
  uint64_t size;
```